Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -22,6 +21,7 @@ namespace OJS.Services.Ui.Business.Implementations
 
     public class SubmissionsBusinessService : ISubmissionsBusinessService
     {
+        private const int DefaultPublicSubmissionsToReturn = 15;
```

Comment:
this can be moved in GlobalConstants, we have it similar with default contests count for home page